tracer/src/Datadog.Trace/Configuration/DbmPropagationLevel.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="DbmPropagationLevel.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.SourceGenerators;
+
+namespace Datadog.Trace.Configuration
+{
+    /// <summary>
+    /// Gets or sets a value indicating whether the tracer should propagate service data in db queries
+    /// </summary>
+    [EnumExtensions]
```

Comment:
Are you using the extension methods anywhere? If not, I recommend deleting it so we can eliminate unused code